core-9/src/main/java/configuration/car/FordConveyorB.java

```diff
@@ -1,14 +1,20 @@
 package configuration.car;
 
-public class FordConveyorB implements FordConveyor {
-    Car car;
+import configuration.car.carImpl.FordBronco;
+import org.springframework.beans.BeansException;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.ApplicationContextAware;
 
-    public FordConveyorB(Car car) {
-        this.car = car;
-    }
+public class FordConveyorB implements FordConveyor, ApplicationContextAware {
```

Comment:
okay, I will count the first conveyor, but this is still the same, I need another Spring-based way to create prototype beans